hyperspy/drawing/signal1d.py

```diff
@@ -310,7 +307,6 @@ def plot(self, data=1):
                                      fontsize=12,
                                      color=self.line.get_color(),
                                      animated=self.ax.figure.canvas.supports_blit)
-        self.ax.figure.canvas.draw_idle()
```

Comment:
Doesn't removing this imply that the line is not drawn when calling the ``plot`` method? If yes, why would that be a good idea?